### PR TITLE
[MRG] some minor paper comments

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

File: content/03.results.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ This creates a gradient of similarity from genus to superkingdom.
 Path selection using the representative genomes in GTDB rs202 resulted in 4095 paths comprised of 9213 unique genomes (8790 Bacteria, 333 Archaea).
 These paths include genome comparisons across 40 phyla (36 Bacteria, 4 Archaea), covering roughly a quarter of the 169 phyla (149 Bacteria, 20 Archaea) in GTDB release rs202.
 While paths are limited to taxonomies with at least two GTDB representative genomes for each taxonomic rank, these paths provide a rich resource for comparisons at increasing evolutionary distances.
+(CTB: are these available for download?)
```
Comment (bluegenes) [inline content/03.results.md:52]: Protein databases are not yet downloadable - we should make them available prior to or simultaneously with preprinting. Are you suggesting I also make an "evolpaths" database with just the genomes selected for this analysis. That's now very doable with sig extract --> zip formats, can do.

As we make protein db's available, we will also need to add documentation about how and when to use protein search (and/or just link this paper)

File: content/03.results.md [hunk 2]
```diff
@@ -71,7 +72,7 @@ While DNA k-mers can provide resolution at the genus level, log-transformed cont
 Average containment estimated from proteome sequence is very similar to anchor containment estimated from 6-frame translation of genome sequence, suggesting that either value can be used for this type of comparison.
 We obtained similar results when comparing all available k-mers, suggesting that these results are not affected by FracMinHash scaling (_Supplemental Figure XX_).
 
-![**Protein k-mers are shared at higher taxonomic ranks** Default scaled values 1000, 200](images/gtdb-rs202.evolpaths.alphacompare.avgContain.logscale.ridgeplot.png){#fig:evolpaths-lca-containment height=4in}
+![**Protein k-mers are shared at higher taxonomic ranks** Default scaled values 1000, 200. CTB: provide log scale on x axis.](images/gtdb-rs202.evolpaths.alphacompare.avgContain.logscale.ridgeplot.png){#fig:evolpaths-lca-containment height=4in}
```
Comment (bluegenes) [inline content/03.results.md:75]: The x is displayed in log - how were you thinking of modifying?

**Review by bluegenes — APPROVED** [review]

File: content/03.results.md (hunk 2 above)
Comment (ctb) [reply content/03.results.md:75]: it needs to be indicated on the axis that it's log, I think? It's obvious when you look at the numbers themselves.

File: content/03.results.md (hunk 1 above)
Comment (ctb) [reply content/03.results.md:52]: either directly available, or perhaps as picklists! how exciting! 😆 

I think making them available for this paper is first thing, then we can (at same time, or later) write documentation.

**Comment by ctb** [comment]
updated from main; and also added a small header-level fix in b5d0997.

**Pull request merged**